components/Dropdown/dropdown.scss

```diff
@@ -29,6 +28,7 @@
   padding: 0 18px 0 10px;
   font-size: 19px;
   margin-right: auto;
+  border-left: solid white 1px;
```

Comment:
by moving the border to the case load the line won't be displayed if there isn't a case load